DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml

```diff
@@ -2,59 +2,73 @@
+                                if (Model.Section.SingleSelectOnly == true)
+                                {
+                                    <div class="filter-option govuk-radios__item">
+                                        @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}", @class = "govuk-radios__input" })
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
+                                else if (Model.Section.SectionDataType == "TrainingRoute")
+                                {
+                                    <div class="filter-option govuk-radios__item">
+                                        @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}", @class = "govuk-radios__input" })
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
+                                else
+                                {
+                                    <div class="filter-option govuk-checkboxes__item">
+                                        @Html.CheckBoxFor(m => Model.Section.Options[jj].IsSelected, new { @class = (Model.Section.Options[jj].ClearOtherOptionsIfSelected ? "filter-na govuk-checkboxes__input" : "govuk-checkboxes__input") })
+                                        <label class="govuk-label govuk-checkboxes__label" for="Section_Options_@jj.ToString()__IsSelected">@Model.Section.Options[jj].Name<span> @Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
 
-                            @Html.HiddenFor(m => Model.Section.Options[jj].OptionKey)
-                            @Html.HiddenFor(m => Model.Section.Options[jj].ClearOtherOptionsIfSelected)
+                                @Html.HiddenFor(m => Model.Section.Options[jj].OptionKey)
+                                @Html.HiddenFor(m => Model.Section.Options[jj].ClearOtherOptionsIfSelected)
+                            }
                         }
-                    }
+                    </div>
                 </div>
-                <button type="submit" class="button filter-next" id="filter-continue">Continue</button>
-            </div>
-            </form>
+
+
+                <button type="submit" class="govuk-button filter-next" id="filter-continue">Continue</button>
+                <a href="#" class="govuk-link govuk-!-margin-left-5">Skip this step</a>
```

Comment:
Don't think # is correct for "skip", this needs to be correctly coded

Reply:
Skip this step was removed as agreed. This cant be just a URL and it must be a submit button which isn't possible when the Continue is also a submit button